contentreview-impl/impl/src/java/org/sakaiproject/contentreview/impl/turnitin/TurnitinContentValidator.java

```diff
@@ -157,17 +157,21 @@ public boolean isAcceptableContent(ContentResource resource) {
 		if (!fileTypeOk) {
 			return false;
 		}
-
-		//TODO: if file is too big reject here 10.48576 MB
+		else {
+			return true;
+		}
+	}
+	
+	public boolean isAcceptableSize(ContentResource resource) {
 
 		if (resource.getContentLength() > tii_Max_Fil_Size) {
-			log.debug("File is too big: " + resource.getContentLength());
+			log.warn("File is too big: " + resource.getContentLength());
```

Comment:
Should include the if of the ContentResource so that we can track down what's causing these.